app/javascript/packs/card.js

```diff
@@ -1,47 +1,42 @@
+let modelSelected = document.getElementById("model-select");
```

Comment:
Nowadays, ordinally there is no case to write many Javascript without any frontend framework like Vue, React... 
Because it's really hard to keep simplicity, readability and maintainability of the code.

In real development, you will face the situation that PM ask you to superior UI. In that case, please stop and consider "Shouldn't it be developed by frontend?, or is it really necessary interface?" 

There are two different types of applications in Raksul.
**Web** for public user and **Ops** for internal operators.

For public user's application, we're often required to maximize the user interface. Then, like this Javascript front interface are necessary and it's maintained with Vue.js.
On the other hand, introducing the frontend framework requires learning cost, too. Then, for OPS, we develop the interface with marginal usability and many time software engineer should communicate PM to minimize the development cost and take much more development time for public user's application.

In conclusion, if you will develop like this operator's form, you may want to develop simpler even if the interface isn't fashionate.

Reply:
About technical, I understand how hard to read my JS code and my form will be much more simpler if manager register inventory one by one (validation and creating may be simpler too).

But after the discussion with Nhan san during step 1 (wireframes), I think register multiple inventories is not a bad idea at all. When I asked Nhan san about import CSV data to register multiple inventories, he told me to think about is it really necessary for manager to make a CSV file with a few records. IMO, this application is make for used phone shops and it is not a big number of phone's types when the shop owner (manager) import these phone daily, weekly or monthly.

So anyway, registration for multiple inventories at once makes me interested and I tried to do this (cause I'm not similar with FE framework so JS is the comfortable way that help me create this feature). It must be easier if I do it one by one or using CSV to import an amount of phones.

Reply:
That makes sense!
So, in real situation, let's communicate Product Manager about the user's requirement well.